app/main/views/marketplace.py

```diff
@@ -172,19 +173,15 @@ def list_opportunities(framework_slug):
     api_result = form.get_briefs()
 
     briefs = api_result["briefs"]
-    links = api_result["links"]
+    meta = api_result['meta']
 
-    api_prev_link_args = parse_link(links, "prev")
-    prev_link_args = None
-    if api_prev_link_args:
-        prev_link_args = request.args.copy()
-        prev_link_args.setlist("page", api_prev_link_args.get("page") or ())
+    results_per_page = meta.get('per_page')
```

Comment:
dictionary['foo'] is the same as dictionary.get('foo'), but you're using both in this code block.  I suggest sticking with [] syntax until you need the default value (like below with requests.args.get('page', 1)